package.json

```diff
@@ -28,15 +28,14 @@
   },
   "peerDependencies": {
     "defu": "^6.1.0",
-    "h3": "^0.8.6"
+    "h3": "^1.0.1"
   },
   "devDependencies": {
     "@nuxtjs/eslint-config-typescript": "^11.0.0",
     "@types/node": "^18.11.0",
     "@vitest/coverage-c8": "^0.25.1",
     "defu": "^6.1.0",
     "eslint": "^8.27.0",
-    "h3": "^0.8.6",
```

Comment:
Would you keep h3 as a dev dependency too?
Otherwise, Vitest can't import its methods in unit test files.

After restoring, please run `pnpm install --force` to update `pnpm-lock.yaml` again.

Reply:
My bad, I didn’t mean to remove it in the first place. 
Added it back with the new commit